src/test/groovy/org/prebid/server/functional/tests/BaseSpec.groovy

```diff
@@ -89,4 +90,12 @@ abstract class BaseSpec extends Specification implements ObjectMapperWrapper {
             }]
         }
     }
+
+    protected static Map<String, List<BidderRequest>> getRequest(AmpResponse ampResponse) {
+        ampResponse.ext.debug.bidders.collectEntries { bidderName, bidderCalls ->
```

Comment:
You duplicate most of the code in both methods. Please extract identical code to a separate private method.